doc/source/rllib/new-api-stack-migration-guide.rst

```diff
@@ -22,29 +22,21 @@ RLlib classes and code to RLlib's new API stack.
     Note, though, that the Ray team continues to  design RLlib to be framework-agnostic.
 
 
-Change your AlgorithmConfig
----------------------------
+Check your AlgorithmConfig
+--------------------------
 
-RLlib turns on the new API stack by default for all RLlib algorithms. To deactivate it, use the `api_stack()` method
-in your `AlgorithmConfig` object like so:
+RLlib turns on the new API stack by default for all RLlib algorithms.
```

Comment:
Didn't I change this already in here: https://github.com/ray-project/ray/pull/49304 ?

Reply:
Yeah, but I removed the "to deactivate it (<- the new API stack), ...". I think we shouldn't describe to users how to deactivate the new API stack on the migration guide site. It's too confusing. Basically, we are saying now: "It's already on, don't worry! But you can double-check your config by doing xyz ..."